xml/decoders/Digitrax_Kato.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file for a specific decoder or decoder family? If so, it might be good to give the file a more specific name, for example Digitrax_Kato_EM13.xml.

Reply:
It was a provision in case there are others, but I think that EM13 is the only one. So renamed the file as suggested.
Note that older versions of the EM13 have a Kato manufacturer code (40) whereas the newest are full fledged Digitrax SW (hence this new definition).